src/graphql/helpers.ts

```diff
@@ -165,6 +165,21 @@ export async function fetchSpaces(args) {
   orderDirection = orderDirection.toUpperCase();
   if (!['ASC', 'DESC'].includes(orderDirection)) orderDirection = 'DESC';
 
+  if (where.strategies_in) {
+    queryStr += " AND JSON_OVERLAPS(JSON_EXTRACT(settings, '$.strategies[*].name'), JSON_ARRAY(?))";
```

Comment:
It only returns spaces that return space that uses only one strategy, it should return any space that use the strategy being passed

For example if space A is using `ticket` and `erc721` and space B is using just `ticket` strategy,
now if we request with `strategies_in:["ticket"]` it return only space B, it should return space A too

Reply:
It should be the case already, that's the job of `JSON_OVERLAPS`. Do you have a specific example I can test ? 

Running the query in the description is giving me spaces using multiple strategies

```graphql
"strategies": [
          {
            "name": "whitelist"
          },
          {
            "name": "delegation"
          },
          {
            "name": "moloch-all"
          },
          {
            "name": "loot-character-guilds"
          },
          {
            "name": "rocketpool-node-operator"
          },
          {
            "name": "honeyswap"
          },
          {
            "name": "xdai-stake-holders"
          },
          {
            "name": "yearn-vault"
          }
        ],
```